src/inmanta/module.py

```diff
@@ -1657,7 +1657,7 @@ class ProjectMetadata(Metadata, MetadataFieldRequires):
         Annotated[str, StringConstraints(strip_whitespace=True, pattern=_re_relation_precedence_rule, min_length=1)]
     ] = []
     strict_deps_check: bool = True
-    agent_install_dependency_modules: bool = False
+    agent_install_dependency_modules: bool = True
```

Comment:
Do we even still need the option? I think we can just drop it completely? As long as we don't break models that set it, but I think extra options are always just ignored.